instrumentation/log4j/log4j-2.13.2/library-autoconfigure/build.gradle.kts

```diff
@@ -3,16 +3,11 @@ plugins {
 }
 
 dependencies {
```

Comment:
Should this module include `implementation(project("...:log4j-2.13.2:library"))`?

Reply:
this case is a bit odd. the `autoconfigure` instrumentation doesn't enable the appender (`library`) instrumentation. another option is to rename `library-autoconfigure` to `library-context-data`.... I'll open an issue to discuss and track